platform/security/platform-security-core-api/src/main/java/ddf/security/common/audit/SecurityLogger.java

```diff
@@ -13,235 +13,275 @@
-     * @param message CXF Message containing the SAML assertion.
+     * @param message the message string to log.
+     * @param subject the user subject to log
      */
-    public static void logSecurityAssertionInfo(Message message) {
-        if (message != null) {
-            SecurityToken token = getToken(message);
-            String requestLogInfo = requestIpAndPortMessage(message);
-
-            // grab the SAML assertion associated with this Message from the
-            // token store
-            if (token != null) {
-                LOGGER.info("SAML assertion successfully extracted from incoming Message.{}",
-                        requestLogInfo);
-                logSecurityAssertionInfo(token);
-            } else {
-                LOGGER.info("No SAML assertion exists on the incoming Message.{}", requestLogInfo);
-            }
-        }
+    public static void audit(String message, Subject subject) {
+        String ipAndPort = requestIpAndPortAndUserMessage(subject,
+                PhaseInterceptorChain.getCurrentMessage());
+        LOGGER.info(ipAndPort + message);
     }
 
-    public static void logSecurityAssertionInfo(SecurityToken token) {
-        if (LOGGER.isDebugEnabled() && token != null) {
-            LOGGER.debug(getFormattedXml(token.getToken()));
-        }
+    /**
+     * Logs a message object with the {@link Level#INFO INFO} level.
+     *
+     * @param message the message string to log.
+     */
+    public static void audit(String message) {
+        String ipAndPort = requestIpAndPortAndUserMessage(
+                PhaseInterceptorChain.getCurrentMessage());
+        LOGGER.info(ipAndPort + message);
     }
 
     /**
-     * Transform into formatted XML.
-     */
-    public static String getFormattedXml(Node node) {
-        DOMImplementation impl = node.getOwnerDocument()
-                .getImplementation();
-        if (null != impl) {
-            Document document = impl.createDocument("", "fake", null);
-            Element copy = (Element) document.importNode(node, true);
-            document.importNode(node, false);
-            document.removeChild(document.getDocumentElement());
-            document.appendChild(copy);
-            DOMImplementation domImpl = document.getImplementation();
-            if (domImpl != null) {
-                DOMImplementationLS domImplLs = (DOMImplementationLS) domImpl.getFeature("LS",
-                        "3.0");
-                if (domImplLs != null) {
-                    LSSerializer serializer = domImplLs.createLSSerializer();
-                    serializer.getDomConfig()
-                            .setParameter("format-pretty-print", true);
-                    return serializer.writeToString(document);
-                }
-            }
-        }
-        return "";
+     * Logs a message with parameters at the {@link Level#INFO INFO} level.
+     *
+     * @param message the message to log; the format depends on the message factory.
```

Comment:
Since `params` applies to the message, may make more sense to have subject, message, params. May want to mention that message uses Log4J replacement syntax, not `String.format()` or something else.